editor/store/selectors.js

```diff
@@ -24,6 +23,7 @@ import createSelector from 'rememo';
 import { serialize, getBlockType, getBlockTypes } from '@wordpress/blocks';
 import { __ } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
+import { moment } from '@wordpress/date';
```

Comment:
So after digging more, I found that these are the only places where we really need this timezone. After some thoughts, I pushed a small commit keeping the usage of this special moment version in this module only to avoid creating too much dependency to WordPress server side. (For instance using `wpmoment` in the components module means the components module have to be updated to depend on the `date` module).